src/Microsoft.VisualStudio.ProjectSystem.Managed/Microsoft.VisualStudio.ProjectSystem.Managed.csproj

```diff
@@ -230,6 +230,9 @@
     <Compile Include="ProjectSystem\Rules\GeneralBrowseObject.cs">
       <DependentUpon>GeneralBrowseObject.xaml</DependentUpon>
     </Compile>
+    <Compile Include="ProjectSystem\Rules\ConfiguredBrowseObject.cs">
+      <DependentUpon>GeneralConfiguredBrowseObject.xaml</DependentUpon>
```

Comment:
Afaict the class inside the .cs file is named after the Rule name inside the xaml file. I chose to have the file name match the class name inside instead of matching the xaml file name.  If you know otherwise, please enlighten me.